app/scripts/directives/overview/serviceGroup.js

```diff
@@ -12,10 +12,35 @@ angular.module('openshiftConsole')
       scope: true,
       templateUrl: 'views/overview/_service-group.html',
       link: function($scope) {
-        // Collapse infrastructure services like Jenkins by default on page load.
-        if (ServicesService.isInfrastructure($scope.service)) {
-          $scope.collapse = true;
-        }
+        var localStorageCollapseKey = function() {
+          var uid = _.get($scope, 'service.metadata.uid');
+          if (!uid) {
+            return null;
+          }
+          return 'collapse/service/' + uid;
+        };
+
+        var wasCollapsed = function() {
+          var key = localStorageCollapseKey();
+          if (!key) {
+            return false;
+          }
+
+          return localStorage.getItem(key) === 'true';
+        };
+
+        var rememberCollapsedState = function() {
```

Comment:
persistCollapsedState would be more clear to me, remember is ambigous to me whether you are storing or recalling